service-registry-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/externalmodel/PostAddress.java

```diff
@@ -0,0 +1,100 @@
+package no.difi.meldingsutveksling.serviceregistry.externalmodel;
+
+import com.google.common.collect.Sets;
+
+public class PostAddress {
+    private String name;
+    private String street1;
+    private String street2;
+    private String street3;
+    private String street4;
+    private String postalCode;
+    private String postalArea;
+    private String country;
+
+    public PostAddress() {
+        /**
+         * needed by json marshallar
+         */
+    }
+
+    public PostAddress(String name, String street1, String street2, String street3, String street4, String postalCode, String postalArea, String country) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00107)